fastlane/Fastfile

```diff
@@ -0,0 +1,62 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  
+  before_all do
+    ENV["SLACK_URL"] = "https://hooks.slack.com/services/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"
+  end
+
+  desc "Deploy a new version to the Google Play"
+  lane :deploy do
+    gradle(task: "clean assembleRelease")
+    # upload_to_play_store
+  end
+  
+  
+  desc "Upload APK to Slack channel"
+  private_lane :upload_to_slack do |options|
+    file_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
+    file_name = file_path.gsub(/\/.*\//,"")
+    token =  ENV["SLACK_API_TOKEN"]
+    channel = ENV["SLACK_CHANNEL"]
+    sh "echo Uploading " + file_name + " to Slack"
+    sh "curl https://slack.com/api/files.upload -F token=\"" + token + "\" -F channels=\"" + channel + "\" -F title=\"" + file_name + "\" -F filename=\"" + file_name + "\" -F file=@" + file_path
+  end  
+  
+  after_all do |lane|
+    # This block is called if executed lane was successful
+    file_name = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH].gsub(/\/.*\//,"")
+    slack(
+      message: "Build Successful!",
+      payload: {
+      # Optional, lets you specify any number of your own Slack attachments.
+      "Build Date" => Time.new.to_s,
+      "App" => file_name,
+      "Built by" => "Tiana"
```

Comment:
`Built by` should house your name and not anyone else's

Reply:
I like testing with my daughter's.. inspiration, it slipped. Thanks